OptimizelySDK/OptimizelyTests/OptimizelyTests-DataModel/ConditionHolderTests_Evaluate.swift

```diff
@@ -162,3 +177,39 @@ extension ConditionHolderTests_Evaluate {
     }
 
 }
+
+// MARK: - Others
+
+extension ConditionHolderTests_Evaluate {
+
+    func testEvaluate_Empty() {
+        let model: ConditionHolder = try! OTUtils.model(from: [])
+        do {
+            _ = try model.evaluate(project: project, attributes: attributeData)
+            XCTAssert(false, "cannot evaluate empty condtion")
+        } catch {
+            XCTAssert(true)
+        }
+    }
+    
+    func testEvaluate_InvalidFirstItem() {
+        let model: ConditionHolder = try! OTUtils.model(from: [["and", userAttributeData]])
+        do {
+            _ = try model.evaluate(project: project, attributes: attributeData)
+            XCTAssert(false, "only operator or leaf node can be the first item")
+        } catch {
+            XCTAssert(true)
+        }
+    }
+    
+    func testEvaluate_OperatorOnly() {
```

Comment:
Where is the error thrown here?  This appears to now hit the  self[1...] where it would throw an exception

Reply:
I guess you expect out-of-bound exception here. It does not happen since the array size is 1 here ( "and" is not filtered out yet).